signingscript/docker.d/passwords.yml

```diff
@@ -97,6 +97,16 @@ in:
              {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
              ["autograph_apk", "autograph_apk_mozillaonline"]
             ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_FOCUS_STAGE_USERNAME"},
+             {"$eval": "AUTOGRAPH_FOCUS_STAGE_PASSWORD"},
+             ["autograph_focus_stage"]
+            ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_FENIX_STAGE_USERNAME"},
+             {"$eval": "AUTOGRAPH_FENIX_STAGE_PASSWORD"},
+             ["autograph_apk_stage", "autograph_apk_mozillaonline_stage"]
+            ]
```

Comment:
I think the plan was to set this up against adhoc right?

Reply:
I recall our conversation about setting it up in both places now, but you'll still need to add this for adhoc.